### PR TITLE
Vp/min 133 helper module for unit tests

### PR DESCRIPTION
**Pull request by dev-vadim**
**Description:**

Moved frequently repeated code from mocks to separate module containing macros.

**Changes in storage or API:**

No changes in storage or API.

### CODE REVIEW

File: pallets/controller/Cargo.toml [hunk 1]
```diff
@@ -23,6 +23,7 @@ minterest-primitives = { path = "../../primitives", default-features = false }
 minterest-model = { path = "../minterest-model", default-features = false }
 liquidity-pools = { path = "../liquidity-pools", default-features = false }
 pallet-traits = { path = "../traits", default-features = false }
+test-helper = { path = "../../test-helper", default-features = false }
```
Comment (lut1k) [inline pallets/controller/Cargo.toml:26]: I think we can use `dev-dependencies` section:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

File: pallets/controller/src/mock.rs [hunk 2]
```diff
@@ -18,6 +18,13 @@ use sp_runtime::{
 	FixedPointNumber, ModuleId,
 };
 use sp_std::cell::RefCell;
+use test_helper::{
+	mock_impl_liquidity_pools_config, mock_impl_orml_currencies_config, mock_impl_orml_tokens_config,
+	mock_impl_system_config,
+};
+
+pub type AccountId = u64;
+type Amount = i128;
```
Comment (lut1k) [inline pallets/controller/src/mock.rs:27]: `type Amount` is only used in orml_tokens. Maybe move this type to the macros for orml_tokens?

File: test-helper/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,127 @@
+//! # Test Helper Module
```
Comment (lut1k) [inline test-helper/src/lib.rs:1]: We have implemented macros for liquidity-pools and liquidation-pools. Why can't we implement macros for the rest of our pallets (dex, controller, minterest-model)?

**Review by lut1k — APPROVED** [review]

**Comment by DenisRomanovsky** [comment]
Looks great to me.

File: pallets/controller/Cargo.toml (hunk 1 above)
Comment (dev-vadim) [reply pallets/controller/Cargo.toml]: Done

File: pallets/controller/src/mock.rs (hunk 2 above)
Comment (dev-vadim) [reply pallets/controller/src/mock.rs]: Done

File: test-helper/src/lib.rs (hunk 3 above)
Comment (dev-vadim) [reply test-helper/src/lib.rs:1]: Done

**Review by lut1k — APPROVED** [review]

**Review by kurotychb — APPROVED** [review]

**Review by Apollo50 — APPROVED** [review]

**Review by lut1k — APPROVED** [review]

**Review by Apollo50 — APPROVED** [review]

**Pull request merged**